doc/distrib/NodeHelpFiles/CoreNodeModels.Input.DoubleInput.md

```diff
@@ -1,5 +1,5 @@
 ## In Depth
-Room numbers are returned as strings.
+The Number node allows a user to enter a static number. This node is useful for things like constants that don't change values. Numbers can also be created by using a Code Block node. In the example below, the number is paired with a code block to create a list sequence with a variable step size.
```

Comment:
thanks for fixing this @twastvedt - we should really fix it in the node markdown docs generation tool which is finding the wrong information from the dictionary - I see that it also happens for a bunch of a other nodes with similar names in different namespaces.

Reply:
Jira issue filed